zetaclient/chains/bitcoin/observer/observer.go

```diff
@@ -240,26 +240,35 @@ func (ob *Observer) WatchRPCStatus(_ context.Context) error {
 
 			bn, err := ob.btcClient.GetBlockCount()
 			if err != nil {
-				ob.logger.Chain.Error().Err(err).Msg("RPC status check: RPC down? ")
+				ob.logger.Chain.Error().Err(err).Msg("RPC status check failed: GetBlockCount")
```

Comment:
NIT : consider breaking up the log line into pieces for example 
```
ob.logger.Chain.Error().
    Err(err).
    Str("context", "RPC status check").
    Str("operation", "GetBlockCount").
    Msg("RPC status check failed")
    ```

Reply:
I think that is reasonable but I'd like to spend more time and do that across the whole zetaclient codebase.